### PR TITLE
Fix macosx runpath dynamic library issue freetype lib - JDK8

### PR DESCRIPTION
**Pull request by neomatrix369**
@smlambert and Muneer came across the same test failure reported via issue #202, see https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/266/testReport/junit/javax_sound_midi_Devices_InitializationHang/java/InitializationHang/

This is due to the fact JDK8 has its JDK and JRE folders named slightly differently than the other JDKs. This has now been corrected.

The build will break if the one of dynamic lib files are not found, there will be a warning for the other one:

libfreetype.6.dylib (error, build breaks)
libfontmanager.dylib (warning, build continues)

Let me know if this flow is good and acceptable.

Please also test this out in other versions of the JDK wherever relevant so we know that the fix works for all MacOSX compliant JDKs.

Scripts have been adapted that it does the respective actions for the different versions of the JDK:
JDK8: only copy to JRE folder
JDK9: copy to JDK and JRE folder
JDK10 onwards: skip this copy process 

@smlambert and Muneer can you please confirm if your tests fail for any of these versions after the fix has been applied. Have you had a chance to test it for JDK10 and JDK11?

### CODE REVIEW

**Comment by neomatrix369** [comment]
Ill be running this against as many JDK builders I can on Jenkins and posting back the links

**Comment by neomatrix369** [comment]
Kicked off the following builds:

https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk8_build_docker_macosx_fix/
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk8_build_x86-64_macos_libfreefont_fix/
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk9_build_x86-64_macos-libfreetype/
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk10_build_x86-64_macos-libfreetype/

Please suggest any others!

File: sbin/build.sh [hunk 1]
```diff
@@ -281,8 +284,21 @@ removingUnnecessaryFiles()
...
+        echo "PERFORM_COPYING=${PERFORM_COPYING}"
+        if [ "${PERFORM_COPYING}" == "false" ]; then
+            echo "${info} Skipping copying of the free font library to ${IMAGE_DIRECTORY}, does not apply for this version of the JDK. ${normal}"
+            return
+        fi
+
+       echo "${info} Performing copying of the free font library to ${IMAGE_DIRECTORY}, applicable for this version of the JDK. ${normal}"
         SOURCE_LIB_NAME="${IMAGE_DIRECTORY}/lib/libfreetype.dylib.6"
+        if [ ! -f "${SOURCE_LIB_NAME}" ]; then
+            echo "${error}[Error] ${SOURCE_LIB_NAME} does not exists in the ${IMAGE_DIRECTORY} folder, please check if this is the right folder to refer to, aborting copy process...${normal}"
```
Comment (karianna) [inline sbin/build.sh]: exists --> exist

File: sbin/build.sh [hunk 2]
```diff
@@ -293,7 +309,13 @@ makeACopyOfLibFreeFontForMacOSX() {
         
         set -x
         cp "${SOURCE_LIB_NAME}" "${TARGET_LIB_NAME}"
-        otool -L "${INVOKED_BY_FONT_MANAGER}"
+        if [ -f "${INVOKED_BY_FONT_MANAGER}" ]; then
+            otool -L "${INVOKED_BY_FONT_MANAGER}"
+        else
+            # shellcheck disable=SC2154
+            echo "${warning}[Warning] ${INVOKED_BY_FONT_MANAGER} does not exists in the ${IMAGE_DIRECTORY} folder, please check if this is the right folder to refer to, this may cause runtime issues, please beware...${normal}"
```
Comment (karianna) [inline sbin/build.sh]: exists --> exist

**Comment by karianna** [comment]
Are we OK to merge this now?

**Pull request merged**